src/downloader/utils.py

```diff
@@ -29,10 +29,15 @@ def download_revanced(config: RevancedConfig, patcher: Patches) -> None:
             ["inotia00", "revanced-integrations", config.integrations_apk],
             ["inotia00", "revanced-patches", config.patches_jar],
         ]
-    if "youtube" in config.apps or "youtube_music" in config.apps:
-        assets += [
-            ["inotia00", "mMicroG", "mMicroG-output.apk"],
-        ]
+    if "youtube" in config.apps or "youtube_music" in config.apps or "MicroG" in config.apps:
```

Comment:
Can you fix the lint [issue](https://results.pre-commit.ci/run/github/524139377/1689338087.Li2XkX_AQoa8zvuAZ8uYcg) by running.

`pip install pre-commit && pre-commit run --all-files`.

Reply:
Done!